applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c

```diff
@@ -247,41 +257,42 @@ static void advertising_process(struct k_work *work)
 	bt_addr_le_t addr;
 
 	if (!k_msgq_get(&bonds_queue, &addr, K_NO_WAIT) && !dir_adv_timed_out) {
-		ret = direct_adv_create(addr);
+		ret = direct_adv_create(ext_adv_index, addr);
```

Comment:
Probably we don't need to change the connection related ADV-EXT?

Reply:
Same answer as above